pom.xml

```diff
@@ -149,7 +149,7 @@ under the License.
 			to revisit the impact at that time.
 		-->
 		<minikdc.version>3.2.0</minikdc.version>
-		<hive.version>2.3.4</hive.version>
+		<hive.version>2.3.7</hive.version>
```

Comment:
As we will also offically provide flink-sql-connector-hive-2.3.9.jar, could you please update it to 2.3.9  and verify it in with jdk 11?